### ISSUE TITLE
#import "../../config.h" Not found

### ISSUE THREAD

**Issue by eshwartm** [peter-iakovlev/Telegram#34]
I cloned the repo and did a build. I got the aforementioned error in `TGShareContextSignal.m` file. Any clues? Should I make a config file as such?

**Comment by dharasisbehera** [comment]
+1

**Comment by albtsantos** [comment]
Yes, you should create your own `config.h` file on the root directory. It should look like:

```
#define SETUP_API_ID(apiId) apiId = 1;
#define SETUP_API_HASH(apiHash) apiHash = @"";
```

You can generate your credentials [here](https://my.telegram.org) (and then accessing “API development tools”).

**Comment by ghost** [comment]
Hi,
Thanks for your solution, I created config.h and config.ini in root of  Telegram folder with apiId and my apiHash. But i still get same error.
Any solution ?
Thanks
